packages/gestalt/src/Link.js

```diff
@@ -14,26 +20,83 @@ import useFocusVisible from './useFocusVisible.js';
 import layoutStyles from './Layout.css';
 
 type Props = {|
+  /**
+   * Supply a short, descriptive label for screen-readers to replace link texts that don't provide sufficient context about the link component behavior. Texts like `Click Here,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the link text. It populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of the link text. See the [Accessibility](https://gestalt.pinterest.systems/link#accessibility) guidelines for more information.
+   */
   accessibilityLabel?: string,
+  /**
+   * Use accessibilitySelected and role when using it as a Tab. See the [Accessibility](https://gestalt.pinterest.systems/link#Accessible-Tab-Link) guidelines for more information.
+   */
   accessibilitySelected?: boolean,
+  /**
+   * Link is a wrapper around components (or children), most commonly [Text](https://gestalt.pinterest.systems/text), so that they become hiperlinks. See the [Link and Text](https://gestalt.pinterest.systems/link#Link-and-Text) variant for more information.
+   */
   children?: Node,
+  /**
+   * When `undeline` is supplied, Link's text is underlined on hover. See the [tapStyle and hoverStyle](https://gestalt.pinterest.systems/link#tapStyle-and-hoverStyle) variant for more information.
+   */
   hoverStyle?: 'none' | 'underline',
+  /**
+   * The URL that the hyperlink points to.
+   */
   href: string,
+  /**
+   * Id attribute of the anchor tag
+   */
   id?: string,
+  /**
+   * Properly positions Link relative to an inline element, such as [Text](https://gestalt.pinterest.systems/text), using the inline property.
+   */
   inline?: boolean,
+  /**
+   * Callback triggered when the user blurs Link.
```

Comment:
Perhaps "…when the element loses focus" — the user isn't always the one shifting focus

Reply:
DONE